Scripts/Insights_19.10.3_hotfix.ps1

```diff
@@ -0,0 +1,52 @@
+﻿param(
+   [Parameter(Mandatory=$true)][string]$username
+  ,[Parameter(Mandatory=$true)][string]$password
+)
+
+function Get-checkBuild($tenant)
+{
+        $cubeBuilding = 0
+        while ($cubeBuilding -ne 1)
```

Comment:
should we have a timeout here (something big but preventing from infinite running)?

Reply:
Will throw an error after 30m of waiting for a build to complete and script will terminate. 

                throw 'Cube is still building after 30m. Please execute script during downtime.'